Java heap space: java.lang.OutOfMemoryError

https://github.com/open-learning-exchange/myplanet/actions/runs/7505690137/job/20435487183#step:8:479
```
FAILURE: Build completed with 2 failures.
1: Task failed with an exception.
For more on this, please refer to https://docs.gradle.org/8.2/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
-----------
56 actionable tasks: 56 executed
* What went wrong:
Execution failed for task ':app:packageReleaseBundle'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.PackageBundleTask$BundleToolWorkAction
   > Java heap space
* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org./
==============================================================================

2: Task failed with an exception.
-----------
* What went wrong:
Execution failed for task ':app:packageRelease'.
> A failure occurred while executing com.android.build.gradle.tasks.PackageAndroidArtifact$IncrementalSplitterRunnable
   > java.lang.OutOfMemoryError (no error message)
* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org./
==============================================================================

BUILD FAILED in 4m 10s
Error: Process completed with exit code 1.
```

--- a/app/build.gradle
+++ b/app/build.gradle
@@ -9,8 +9,8 @@ android {
         applicationId "org.ole.planet.myplanet"
         minSdkVersion 21
         targetSdkVersion 34
-        versionCode 1279
-        versionName "0.12.79"
+        versionCode 1291
+        versionName "0.12.81"
         ndkVersion '21.3.6528147'
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
         vectorDrawables.useSupportLibrary = true

--- a/gradle.properties
+++ b/gradle.properties
@@ -9,8 +9,7 @@
 
 # Specifies the JVM arguments used for the daemon process.
 # The setting is particularly useful for tweaking memory settings.
-#org.gradle.jvmargs=-Xmx1536m
-org.gradle.jvmargs=-Xms512m -Xmx1024m --add-opens=jdk.compiler/com.sun.tools.javac.code=ALL-UNNAMED
+org.gradle.jvmargs=-Xms512m -Xmx4g --add-opens=jdk.compiler/com.sun.tools.javac.code=ALL-UNNAMED
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects

